packages/block-library/src/navigation/use-api-fetch.js

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
I added the reusable apiFetch hook in the navigation folder as I'm not sure what the best place for it would be, or where it might be reused. Feedback appreciated!

Reply:
Hmm, I suppose if it went in the api-fetch package, react would have to be added as a dependency to that.

Reply:
What do you think, @youknowriad? Adding `@wordpress/element` as a dependency of `@wordpress/api-fetch` isn't totally out of the question since e.g. `@wordpress/data` depends on `@wordpress/element`.

Reply:
Yeah, maybe it's fine. It does increase bundle size if loaded on the frontend without react need but I don't have a better alternative here.